### DISCUSSION TITLE
雖然ChatGPT用不了 但OpenAI還是有其他AI模型能用

### DISCUSSION THREAD

**Discussion by xCk27x** [1101-GS3073-C/Python-Community-GS3073-C#673] [General]
### 雖然ChatGPT用不了 但OpenAI還是有其他AI模型能用

- 因為最近被JavaScript的callback function搞的很頭痛 所以用了OpenAI的 [**JavaScript helper chatbot**](https://beta.openai.com/playground/p/default-js-helper?model=code-davinci-002) 找答案

慣例先上圖
<img width="854" alt="螢幕擷取畫面_20230109_043128" src="https://user-images.githubusercontent.com/90547641/211217958-e526fd7e-fb07-48b1-8f8d-d6540cb2da6b.png">

- 一樣是問問題就會回答，但對中文的理解程度好像沒有到那麼高
- 可以一直在前面問題的基礎上更細節的繼續問下去 省去很多找網路資源的時間

**感覺未來使用AI找答案會是主流 要開始習慣這種模式了**
**OpenAI還有很多特定的AI訓練模型 大家可以多去[https://beta.openai.com/](https://beta.openai.com)玩玩看~**

**Comment by bww0401** [comment]
感謝分享，有了AI找答案，就可以省下更多時間了

**Comment by skyling0299** [comment]
openAI有一些提問技巧啦，他就像是code一樣，你要給他一些正確的明確的指示，他才會給你正確的東西
文法不是必需的 甚至可以條列

**Comment by lx3211** [comment]
之前在做期末小組報告的時候，chatGPT因為沒有官方API，如果從爬蟲來使用它的話，常常會被擋住，我們當時就找了openAI出的在chatGPT以前的AI對話模型來接，但是真的差太多了，chatGPT會大火真的不是沒有原因的
